programs/bpf/c/sdk/bpf.mk

```diff
@@ -50,11 +50,13 @@ help:
 	@echo '  - Output files will be placed in the directory: $(OUT_DIR)'
 	@echo ''
 	@echo 'User settings'
-	@echo '  - The following setting are overridable on the command line, default values shown'
+	@echo '  - The following setting are overridable on the command line, default values shown:'
 	@echo '    - Show commands:'
 	@echo '      V=1'
 	@echo '    - List of include dirs:'
 	@echo '      INC_DIRS=$(INC_DIRS)'
+	@echo '    - List of systme include dirs:'
+	@echo '      INC_DIRS=$(SYSTEM_INC_DIRS)'
```

Comment:
Why add this?

Reply:
So folks can see what's going on

Reply:
But INC_DIRS is not equal to SYSTEM_INC_DIRS.

Reply:
heh, right